microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -721,8 +719,31 @@ Status:
 Controls whether to replay input events on the Timeline.
 
 Status:
-*  This checkbox is present in Microsoft Edge Canary 109.
-*  This checkbox is present in Microsoft Edge Stable 107.
+*  This checkbox is present in Microsoft Edge Canary 111.
+*  This checkbox is present in Microsoft Edge Stable 108.
+
+
+<!-- ====================================================================== -->
+## View console.profile() results in the Performance panel for Node.js
+
+_new as of v109_
+
+Displays `console.profile()` results in the **Performance** tool for Node.js.
+
+`console.profile()` can be used in the DevTools **Console** (or in your code) to start a JavaScript profile, and then you can use `console.profileEnd()` to terminate it.  Once done, you can see your profile recording in the **JavaScript Profiler** tool.
+
+This is a programmatic way to record JavaScript execution, which can be useful in certain performance debugging situations.
+
+Also, DevTools can optionally be used to debug processes that are running in Node.js.  You need to start Node.js in a special way so that it exposes debugging info on a port.  Then you can start DevTools from `edge://inspect`, to target that port.
```

Comment:
Needs to be rephrased. We should either explain how to debug Node.js entirely, or not say anything at all. I would lean towards the latter and rephrase: "When DevTools is used to debug processes that are running in Node.js", and then link it to the next paragraph:

"When DevTools is used to debug processes that are running in Node.js this experiment makes what `console.profile` generates available to the **Performance** Tool."

Reply:
replaced above and below para by:
When DevTools is used to debug processes that are running in Node.js, this experiment makes whatever `console.profile` generates available to the **Performance** Tool.